### DISCUSSION TITLE
Bouton exporter journal alarmes depuis Synapps

### DISCUSSION THREAD

**Discussion by JulienFee** [witsa/synapps#731] [Q&A]
Bonjour,
Quelqu'un peut-il m'aider à créer un bouton exporter dans la page journal alarmes depuis Synapps.
Même fonction que sur les anciens synoptiques...

Merci

**Comment by JeremieDENAEGEL** [comment]
UP
Est-ce-quelqu'un aurait réussi à faire le bouton à partir du journal des reflets ?

**Comment by Marenas8888** [comment]
Bonjour

Des nouvelles concernant cette fonctionnalité ?

Merci en avancé
